fluent/syntax/ftlstream.py

```diff
@@ -73,20 +73,31 @@ def take_char(self, f):
             return ch
         return None
 
-    def is_id_start(self):
-        if self.ch is None:
+    def is_char_id_start(self, ch=None):
```

Comment:
I didn't get to actually look at this patch in full, but this one triggered my early-morning brain. `ch` should be an unconditional argument.